packages/lit-dev-content/src/components/algolia-search-controller.ts

```diff
@@ -18,7 +18,8 @@ const agloliaSearchControllerDefaultOptions = {
   appId: publicVars.algolia.appId,
   searchOnlyKey: publicVars.algolia.searchOnlyKey,
   index: publicVars.algolia.index,
-  hitsPerPage: 20,
+  hitsPerPage: 10,
+  distinct: 4 as number | boolean,
```

Comment:
Is this type assertion required?

Reply:
yes, because distinct can be false. We will be doing this in the SERP PR, and we're inferring the type of the constructor's `options` argument from this variable

Reply:
In the future, perhaps this whole default options object can be asserted with `satisfies`.
https://devblogs.microsoft.com/typescript/announcing-typescript-4-9-beta/#hamilton

(also great that the hash link for `satisfies` operator section is `#hamilton`)